backend/tracim_backend/models/context_models.py

```diff
@@ -1526,3 +1547,13 @@ def filename(self) -> str:
         :return: complete filename with both label and file extension part
         """
         return core_convert_file_name_to_display(self.revision.file_name)
+
+
+class PaginatedObject(object):
+    def __init__(self, page: Page):
```

Comment:
Don't forget to add the typing for the returned value `-> None`.

Reply:
done.